site/app/controllers/admin/SqlToolboxController.php

```diff
@@ -24,7 +24,19 @@ class SqlToolboxController extends AbstractController {
      * @Route("/courses/{_semester}/{_course}/sql_toolbox", methods={"GET"})
      */
     public function showToolbox(): WebResponse {
-        return new WebResponse(SqlToolboxView::class, 'showToolbox');
+        $this->core->getCourseDB()->query("SELECT * FROM information_schema.columns WHERE table_schema='public'");
+        $tables = $this->core->getCourseDB()->rows();
+        $organizedTables = [];
+        //Loop through and create a 2d array that holds all columns for each table name.
+        foreach ($tables as $table) {
```

Comment:
Since we don't use `$tables` anywhere except in the foreach, better to avoid defining it altogether.
```suggestion
        $organizedTables = [];
        //Loop through and create a 2d array that holds all columns for each table name.
        foreach ($this->core->getCourseDB()->rows() as $table) {
```